src/dbnode/server/server.go

```diff
@@ -147,6 +147,10 @@ type RunOptions struct {
 	// interrupt and shutdown the server.
 	InterruptCh <-chan error
 
+	// ShutdownCh is an optional channel to supply if interested in receiving
```

Comment:
What line length are we using? Is this set in gofmt?

Reply:
I'm using 80 in my IDE, but I think the linter only actually dings you at 100 or 120? Don't quite remember.

Reply:
it's enforced by the linter. I think 100. keep in mind the linter was turned on recently (6 months ago?), so a lot of legacy code does not comply.